recipes/libftdi/all/conanfile.py

```diff
@@ -0,0 +1,51 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class LibFtdiConan(ConanFile):
+    name = "libftdi"
+    description = "A library to talk to FTDI chips"
+    license = ["MIT", "BSD-3-Clause"]
+    topics = ("conan", "libftdi")
+    homepage = "http://developer.intra2net.com/git/?p=libftdi;a=summary"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    _source_subfolder = "source_subfolder"
+
+    _cmake = None
+
+    revisions = {"1.4": "d5c1622",
+                 "1.3": "96d337a",
+                 "1.2": "0b6f220"}
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.revisions[self.version]
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.definitions["CMAKE_INSTALL_PREFIX"] = "."
+        cmake.configure(source_dir=self._source_subfolder)
+        return cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
+        self.copy(pattern="*.so", dst="lib", keep_path=False, symlinks=False)
```

Comment:
cmake should also install the shared libraries
```suggestion
```

Reply:
#### Done

`CMAKE_INSTALL_PREFIX` set to `package_folder`.